engine/crates/engine/src/registry/resolvers/postgres/request/create_many.rs

```diff
@@ -1,14 +1,20 @@
+use futures_util::TryFutureExt;
+use grafbase_sql_ast::renderer::{self, Renderer};
+
 use super::{log, query, RowData};
 use crate::registry::resolvers::{postgres::context::PostgresContext, ResolvedValue};
-use grafbase_sql_ast::renderer::{self, Renderer};
-use postgres_types::transport::Transport;
 
 pub(crate) async fn execute(ctx: PostgresContext<'_>) -> Result<ResolvedValue, crate::Error> {
     let input = ctx.create_many_input()?;
     let (sql, params) = renderer::Postgres::build(query::insert::build(&ctx, input)?);
 
     if ctx.mutation_is_returning() {
-        let operation = ctx.transport().parameterized_query::<RowData>(&sql, params);
+        let operation = ctx.transport().parameterized_query(&sql, params).map_ok(|values| {
+            values
+                .into_iter()
+                .map(|value| serde_json::from_value::<RowData>(value).expect("should deserialize to RowData"))
+                .collect()
+        });
```

Comment:
Could you use that map_result function here?

Reply:
oh definitely, left overs